### PR TITLE
IGAPP-656: Move route sprungbrett

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web-ts/src/components/__tests__/SprungbrettListItem.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```
Comment (Taggotty) [inline web-ts/src/components/__tests__/SprungbrettListItem.spec.tsx:1]: replaced snapshot test

File: web-ts/src/components/__tests__/SprungbrettListItem.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  const job = new SprungbrettJobModel({
+    id: 0,
+    title: 'WebDeveloper',
+    location: 'Augsburg',
+    isEmployment: true,
+    isApprenticeship: true,
+    url: 'http://awesome-jobs.domain'
+  })
+
+  it('should render and match snapshot', () => {
```
Comment (klinzo) [inline web-ts/src/components/__tests__/SprungbrettListItem.spec.tsx:18]: adjust test name

File: web-ts/src/routes/SprungbrettOfferPage.tsx [hunk 3]
```diff
@@ -12,25 +38,82 @@ type PropsType = {
...
+  if (offersLoading || sprungbrettLoading) {
+    return (
+      <LocationLayout isLoading {...locationLayoutParams}>
+        <LoadingSpinner />
+      </LocationLayout>
+    )
+  }
+
+  if (!sprungbrettJobs || !offer) {
+    const error = sprungbrettError || offersError || new Error('Sprungbrett and Offers should be available!')
```
Comment (klinzo) [inline web-ts/src/routes/SprungbrettOfferPage.tsx:97]: I think there should be an offer not found error if the sprungbrett offer is not enabled.

**Review by klinzo — APPROVED** [review]
Tested

**Review by maxammann — APPROVED** [review]

**Pull request merged**